mailbox/tools/pom.xml

```diff
@@ -32,6 +32,7 @@
     <name>Apache James :: Mailbox :: Tools</name>
 
     <modules>
+        <module>copier</module>
```

Comment:
we didn't use nested modules for mailbox, it's not very consistent with existing code

Reply:
If you want.

It makes no differences to me.

For the record, is there some rationals to not do so in some cases?

BTW isn't backup a tool?

Reply:
For the record https://github.com/linagora/james-project/pull/1551 `JAMES-2468 Handle quota update on mailbox deletion ` is the root cause

Reply:
> For the record, is there some rationals to not do so in some cases?

less files (and less work for maven)
I don't see other arguments